deployment/infrastructure/Pulumi.dev.yaml

```diff
@@ -1,3 +1,9 @@
 config:
   aws:region: eu-central-1
-  rasporedar:sshPublicKey: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGHeS9LRxPeUFrTL/80ucq4tyYwadWBw45wRdJpPh9+N bdeak@extensionengine.com
+  rasporedar:ghcrPassword:
+    secure: AAABANK3OC5Sq3yNqkLMnrbkMewpyN7s9yVhNUdDEsjG3d3bgJOdN5+70q6wIh6cwUkUY0WS5x0i5FjsdtoKW0GqNLO3L3cq
+  rasporedar:ghcrUsername: bdeak4
+  rasporedar:jwtSecretKey: E05B6878B9119EA79F28F427BF8CDB2E184B09B5578E3D581DE9994221FA6F5E
+  rasporedar:saltRounds: "10"
```

Comment:
Can some of these be passed through environment variables? I know this is dev configuration but still. A secret is a secret 😄